homeassistant/components/wled/sensor.py

```diff
@@ -27,6 +32,10 @@
         WLEDEstimatedCurrentSensor(entry.entry_id, coordinator),
         WLEDUptimeSensor(entry.entry_id, coordinator),
         WLEDFreeHeapSensor(entry.entry_id, coordinator),
+        WLEDWifiBSSIDSensor(entry.entry_id, coordinator),
+        WLEDWifiChannelSensor(entry.entry_id, coordinator),
+        WLEDWifiRSSISensor(entry.entry_id, coordinator),
```

Comment:
These always really spam the state machine and rarely are needed. Can't we disable them by default? 

I think that the others are also not really that helpful ?

Reply:
They are disabled by default, I have even provided tests for that to make sure...